### PR TITLE
Added Support for multiple return values

### PR DESCRIPTION
**Pull request by SpookyBeverage**

### CODE REVIEW

File: apps/frontend/components/NewExp.js [hunk 1]
```diff
@@ -89,12 +89,31 @@ const InformationStep = ({ form, ...props }) => {
 					<div className='sm:col-span-4'>
 						<input
 							type='text'
+							placeholder='Filename of the CSV that are created by the experiment'
```
Comment (budak7273) [inline apps/frontend/components/NewExp.js:92]:
```suggestion
							placeholder='Name and extension of the output CSV file'
```

File: apps/frontend/components/NewExp.js [hunk 2]
```diff
@@ -89,12 +89,31 @@ const InformationStep = ({ form, ...props }) => {
...
 							{...form.getInputProps('fileOutput')}
 							className='block w-full rounded-md border-gray-300 shadow-sm focus:border-blue-500 focus:ring-blue-500 sm:text-sm'
 						/>
 					</div>
 				</InputSection>
+				<InputSection header={'Result Output'}>
+					<div className='sm:col-span-4'>
+						<input
+							type='text'
+							placeholder='Filename of the experiment results'
```
Comment (budak7273) [inline apps/frontend/components/NewExp.js:102]:
```suggestion
							placeholder='Name and extension of the experiment results file'
```

**Review by budak7273 — CHANGES_REQUESTED** [review]

File: scripts/proof_of_concept/addNumsforConfig.py [hunk 3]
```diff
@@ -9,11 +9,7 @@ def main():
     config.read(configFile)
     x = int(config["DEFAULT"]["x"])
     y = int(config["DEFAULT"]["y"])
-    boo = bool(config["DEFAULT"]["boo"])
-    if boo:
-        print(x+y)
-    else:
-        print(x-y)
+    print(x+y)
```
[the remark this answers is not in the document]
Comment (SpookyBeverage) [reply scripts/proof_of_concept/addNumsforConfig.py:12]: This one was always doing addition, and regardless subtraction is covered in the other files

File: apps/frontend/components/NewExp.js [hunk 4]
```diff
@@ -352,8 +337,10 @@ const NewExp = ({ formState, setFormState, ...rest }) => {
 
 									{/* <div className='h-full flex flex-col space-y-6 py-6 sm:space-y-0 sm:divide-y sm:divide-gray-200 sm:py-0'> */}
 									{status === 0 ? (
```
[the remark this answers is not in the document]
Comment (SpookyBeverage) [reply apps/frontend/components/NewExp.js:339]: To be honest I'm not entirely sure how the array on 324 fully works I just figured out how to add the new thing to it tbh. I can look into changing the state to an enum for status without breaking anything

[the remark this answers is not in the document]
Comment (SpookyBeverage) [reply apps/frontend/components/NewExp.js:339]: Though in my opinion having State as a number is simpler

**Review by budak7273 — APPROVED** [review]

**Pull request merged**